package.json

```diff
@@ -37,11 +37,13 @@
     "eslint": "^6.7.2",
     "express": "^4.17.1",
     "jsdom": "^19.0.0",
+    "node-fetch": "^3.3.2",
     "nodemon": "^2.0.7",
     "redis": "^4.6.10",
     "sharp": "^0.30.1",
     "ts-node": "^10.8.1",
-    "typescript": "^4.7.3"
+    "typescript": "^4.7.3",
+    "viem": "^2.9.15"
```

Comment:
We don't need this package, use the already installed `@ethersproject/address` package instead